pipeline/frontend/yaml/compiler/convert.go

```diff
@@ -98,6 +98,16 @@ func (c *Compiler) createProcess(container *yaml_types.Container, stepType backe
 
 	workingDir = c.stepWorkingDir(container)
 
+	getVariableValue := func(name string) (string, error) {
+		name = strings.ToLower(name)
```

Comment:
we switched back to case sensitive for secrets ... so we should stay consistent

Reply:
I can still see `name = strings.ToLower(name)` in the `getVariableValue` even after merging main, should I still proceed to remove them from variables?

Reply:
let me have a look at the last pulls in this topic ...

Reply:
Any updates on this?

Reply:
yes case sensitive it is